modVM/src/org/aion/vm/exception/VMException.java

```diff
@@ -0,0 +1,8 @@
+package org.aion.vm.exception;
+
+public class VMException extends Exception {
```

Comment:
It would be good to document the fact that these exceptions are thrown only in case of fatal VM errors.

Reply:
updated